docs/src/main/asciidoc/hibernate-orm.adoc

```diff
@@ -11,6 +11,16 @@ include::./attributes.adoc[]
 Hibernate ORM is the de facto standard JPA implementation and offers you the full breadth of an Object Relational Mapper.
 It works beautifully in Quarkus.
 
+== Solution
+
+We recommend that you follow the instructions in the next sections and create the application step by step.
+However, you can go right to the completed example.
+
+Clone the Git repository: `git clone {quickstarts-clone-url}`, or download an {quickstarts-archive-url}[archive].
+
+The solution is located in the `hibernate-orm-quickstart` {quickstarts-tree-url}/hibernate-orm-quickstart[directory].
+There is also a dedicated quickstart for multi-tenancy in the `hibernate-orm-multi-tenancy-quickstart` {quickstarts-tree-url}/hibernate-orm-multi-tenancy-quickstart[directory].
```

Comment:
I would rather add that in the multi tenancy doc section.

Something like: "To see multitenancy into action, you can check out the following quickstart: ..."

Reply:
I wasn't sure whether it was best to put it here or on the multi-tenancy section.
I moved the quickstart to the multitenancy section.